core/app/models/spree/refund.rb

```diff
@@ -53,8 +53,12 @@ def perform!
       credit_cents = money.cents
 
       @perform_response = process!(credit_cents)
+
       log_entries.build(parsed_payment_response_details_with_fallback: perform_response)
 
+      log_entry = payment.log_entries.build(parsed_payment_response_details_with_fallback: perform_response)
+      log_entry.save
```

Comment:
Thanks, @cesartalves! That was something that confused me in the past.

Thoughts on?:

- Using `create` instead of `build` + `save`.
- Even better, using `create!` to detect failures.
- Non-blocking: extract to a comment to better communicate intentions. Like `create_payment_log_entry`.